pkg/controller/elasticsearch/validation/webhook.go

```diff
@@ -76,13 +79,21 @@ func (wh *validatingWebhook) validateUpdate(prev esv1.Elasticsearch, curr esv1.E
 	return ValidateElasticsearch(curr, wh.exposedNodeLabels)
 }
 
+// Handle is called when any request is sent to the webhook, satisfying the admission.Handler interface.
 func (wh *validatingWebhook) Handle(_ context.Context, req admission.Request) admission.Response {
 	es := &esv1.Elasticsearch{}
 	err := wh.decoder.DecodeRaw(req.Object, es)
 	if err != nil {
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
+	// If this Elasticsearch instance is not within the set of managed namespaces
+	// for this operator ignore this request.
+	if len(wh.managedNamespaces) > 0 && !slices.Contains(wh.managedNamespaces, es.Namespace) {
```

Comment:
We have

 https://github.com/elastic/cloud-on-k8s/blob/20f522ff9f5fa04f8ccb22ce19edaa6d044a1aa6/pkg/utils/set/set.go#L9

Not that is necessary to use here but it offers a slightly easier to read containment API something like:

```golang
if managedNamespaces.Has(es.Namespace) {
```

Reply:
I updated it to use this package instead, as it certainly enhances the readability.